### ISSUE TITLE
Installed Portainer Add-on but it does not start

### ISSUE THREAD

**Issue by deltabert** [alexbelgium/hassio-addons#736]
**Portainer Add-on**

- Addon name : Portainer
- Addon version : 2.17.1-2

**Describe the bug**
Installed Portainer Add-on, but it does not start.

I wanted to use pip3 in my HAOS, so i stumbled upon the Fork of the Portainer Add-on. 
I installed the Add-on, disabled the Protection mode in the settings of the Portainer Add-on as discribed on the GitHub page.
Restarted Home Assistant, but when i start the Portainer Add-on, it does not start.

**To Reproduce**

**Full addon log**
This is the info in the Log tab of the Portainer Add-on:
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service base-addon-banner: starting
-----------------------------------------------------------
 Add-on: Portainer
 Manage your Docker environment with ease
-----------------------------------------------------------
 Add-on version: 2.17.1-2
 You are running the latest version of this add-on.
 System: Home Assistant OS 9.5  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2023.2.5
 Home Assistant Supervisor: 2023.01.1
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
s6-rc: info: service base-addon-banner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service base-addon-log-level: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service base-addon-log-level successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/00-aaa_portainer_install.sh
cont-init: info: /etc/cont-init.d/00-aaa_portainer_install.sh exited 0
cont-init: info: running /etc/cont-init.d/00-banner.sh
-----------------------------------------------------------
 Add-on: Portainer
 Manage your Docker environment with ease
-----------------------------------------------------------
 Add-on version: 2.17.1-2
 You are running the latest version of this add-on.
 System: Home Assistant OS 9.5  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2023.2.5
 Home Assistant Supervisor: 2023.01.1
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums
 https://github.com/alexbelgium/hassio-addons
-----------------------------------------------------------
cont-init: info: /etc/cont-init.d/00-banner.sh exited 0
cont-init: info: running /etc/cont-init.d/01-custom_script.sh
[11:24:20] INFO: Execute /config/addons_autoscripts/portainer.sh if existing
[11:24:20] INFO: ... no script found
cont-init: info: /etc/cont-init.d/01-custom_script.sh exited 0
cont-init: info: running /etc/cont-init.d/30-nginx.sh
cont-init: info: /etc/cont-init.d/30-nginx.sh exited 0
cont-init: info: running /etc/cont-init.d/portainer.sh
cont-init: info: /etc/cont-init.d/portainer.sh exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun nginx (no readiness notification)
services-up: info: copying legacy longrun portainer (no readiness notification)
[10:24:22] INFO: Starting Portainer...
s6-rc: info: service legacy-services successfully started
[11:24:22] INFO: Waiting for port 9000 to open...
[10:24:24] INFO: ... password set to homeassistant
[10:24:24] INFO: ... non-addon containers hidden
[10:24:24] INFO: ... portainer launched
2023/02/27 10:24AM INF github.com/portainer/portainer/api/cmd/portainer/main.go:534 > encryption key file not present | filename=portainer
2023/02/27 10:24AM INF github.com/portainer/portainer/api/cmd/portainer/main.go:553 > proceeding without encryption key |
2023/02/27 10:24AM INF github.com/portainer/portainer/api/database/boltdb/db.go:124 > loading PortainerDB | filename=portainer.db
2023/02/27 10:24AM INF github.com/portainer/portainer/api/internal/ssl/ssl.go:80 > no cert files found, generating self signed SSL certificates |
2023/02/27 10:24AM INF github.com/portainer/portainer/api/cmd/portainer/main.go:698 > created admin user with the given password. |
2023/02/27 10:24:26 server: Reverse tunnelling enabled
2023/02/27 10:24:26 server: Fingerprint c6:35:a1:2e:bf:54:e8:2c:c1:a5:33:aa:49:ba:b6:ab
2023/02/27 10:24:26 server: Listening on 0.0.0.0:8000...
2023/02/27 10:24AM INF github.com/portainer/portainer/api/cmd/portainer/main.go:810 > starting Portainer | build_number=27976 go_version=1.19.4 image_tag=linux-arm64-2.17.1 nodejs_version=18.14.2 version=2.17.1 webpack_version=5.68.0 yarn_version=1.22.19
2023/02/27 10:24AM INF github.com/portainer/portainer/api/http/server.go:342 > starting HTTPS server | bind_address=:9443
2023/02/27 10:24AM INF github.com/portainer/portainer/api/http/server.go:327 > starting HTTP server | bind_address=0.0.0.0:9000
[11:24:26] INFO: Starting NGinx...

**Full addon config**

**System**
I am running the following version of HAOS:
Home Assistant 2023.2.5
Supervisor 2023.01.1
Operating System 9.5
Frontend 20230202.0 - latest

- Supervisor version: 
- Host system version:

**Comment by alexbelgium** [comment]
Hi, there is no error in your log. Everything seems to start correctly. How do you arrive at the conclusion that "Installed Portainer Add-on, but it does not start." ? Thanks

**Comment by OldSmurf56** [comment]
Hi
Actually I just installed Portainer yesterday and experiencing the same - it appear to run (showing CPU usage etc) but when I want to open the WebGUI, I get a response stating "Add.on is not running. Please start it first"
I then press the "OK" button and I am returned to the Portainer tab ... and Portainer is no longer running.

I have two separate VM running HA on my ESXi but with different IP addresses - the error occurs on both.

Add-on: Portainer
 Manage your Docker environment with ease
-----------------------------------------------------------
 Add-on version: 2.17.1-2
 You are running the latest version of this add-on.
 System: Home Assistant OS 9.5  (amd64 / qemux86-64)
 Home Assistant Core: 2023.2.5
 Home Assistant Supervisor: 2023.01.1
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------

**Comment by PaulProjects** [comment]
did you deactivate the secure mode (last toggle on the info page)

**Comment by alexbelgium** [comment]
Hi, just pushed a new version with updated logic

**Comment by OldSmurf56** [comment]
@PaulProjects - yes. SSL is deactivated
@alexbelgium  - I have updated (and also updated to the latest HA core) but unfortunately nothing has changed: it looks as if Portainer is started even showing CPU usage, but when I try to open WebUI is states add-on not started. And when I return to the Portainer Info tab, it has indeed stopped.
Btw - I have also tried with three different browsers with the same result.
